templates/todo/projects/csharp-sql/README.md

```diff
@@ -33,123 +21,192 @@ The following prerequisites are required to use this application.  Please ensure
     ```
   - Linux/MacOS:
     ```
-    curl -fsSL https://aka.ms/install-azd.sh | bash 
+    curl -fsSL https://aka.ms/install-azd.sh | bash
     ```
 - [Azure CLI (2.38.0+)](https://docs.microsoft.com/cli/azure/install-azure-cli)
 - [.NET SDK 6.0](https://dotnet.microsoft.com/download/dotnet/6.0) - for the API backend
 - [Node.js with npm (16.13.1+)](https://nodejs.org/) - for the Web frontend
 - [Git (2.36.1+)](https://git-scm.com/)
 
-### Project Folder
+### Quickstart
 
-You will need an empty folder on your computer to house the project files.
+The fastest way for you to get this application up and running on Azure is to use the `azd up` command. This single command will create and configure all necessary Azure resources - including access policies and roles for your account and service-to-service communication with Managed Identities.
 
-1. Open your favorite terminal, create a new folder, and set your current directory to that newly created folder.
+1. Open a terminal, create a new empty folder, and change into it.
+1. Run the following command to initialize the project, provision Azure resources, and deploy the application code.
 
-   ```bash
-   mkdir {your-unique-project-folder-name}
-   cd {your-unique-project-folder-name}
-   ```
+```bash
+azd up --template todo-csharp-sql
+```
 
-## Quickstart
+You will be prompted for the following information:
 
-The fastest possible way for you to get this app up and running on Azure is to use the `az dev up` command.
+- `Environment Name`: This will be used as a prefix for the resource group that will be created to hold all Azure resources. This name should be unique within your Azure subscription.
+- `Azure Location`: The Azure location where your resources will be deployed.
+- `Azure Subscription`: The Azure Subscription where your resources will be deployed.
 
-The `az dev up` command will:
+> NOTE: This may take a while to complete as it executes three commands: `azd init` (initializes environment), `azd provision` (provisions Azure resources), and `azd deploy` (deploys application code). You will see a progress indicator as it provisions and deploys your application.
```

Comment:
Is this necessary? If so, might also be worth mentioning that this template is scaffolding all of the necessary infrastructure (DB, permissions etc; not just hosting an app)...because that's what's actually taking some time.

Reply:
Will be handled via https://github.com/Azure/azure-dev/issues/599